packages/react/src/LogoSuite/LogoSuite.features.stories.tsx

```diff
@@ -356,3 +362,17 @@ export const WithLinks = () => (
     </LogoSuite.Logobar>
   </LogoSuite>
 )
+
+export const WithPngLogos = () => (
```

Comment:
Nit, but png is incidental so would recommend calling this story `WithImgLogos` or `WithRasterLogos`.

Reply:
2eda631fbc103f90b2ebee3d3cc5632b55154065